Carbon/navigation/screens/Home/GetData.js

```diff
@@ -0,0 +1,25 @@
+
+/*
+    This is a function that will get the data from the server and return it
+    The data is not being returned yet.
+*/
+const GetData = async() => {
+  try {
+      const url = 'Commit data for ';
+      console.log("fetching data from " + url); //log to see if if it works
+      const response = await fetch(url); //wait for response
+      const data = await response.json(); //get the data we need
+      console.log(data); //print data for testing
```

Comment:
We'll want to remove this in a future PR

Reply:
Agreed, the connection is temporary